tests/output/InteractionsTest.py

```diff
@@ -4,15 +4,17 @@
 
 from coalib.output.Interactions import fail_acquire_settings
 from coalib.output.printers.LogPrinter import LogPrinter
+from coalib.settings.Section import Section
 
 
 class InteractionsTest(unittest.TestCase):
 
     def test_(self):
         log_printer = LogPrinter(NullPrinter())
+        section = Section("")
         self.assertRaises(TypeError, fail_acquire_settings, log_printer, None)
```

Comment:
ah you need to change this line too, a `TypeError` is raised here from supplying not enough arguments :)
So you need to add the section argument here, then the coverage raises to 100% again :+1: